plugin.js

```diff
@@ -88,53 +88,103 @@ class CspHtmlWebpackPlugin {
    * @param compiler
    */
   apply(compiler) {
-    compiler.plugin('compilation', compilation => {
-      compilation.plugin(
-        'html-webpack-plugin-after-html-processing',
-        (htmlPluginData, compileCb) => {
-          const $ = cheerio.load(htmlPluginData.html, {
-            decodeEntities: false
-          });
-
-          // if not enabled, remove the empty tag
-          if (!this.isEnabled(htmlPluginData)) {
-            $('meta[http-equiv="Content-Security-Policy"]').remove();
+    if (compiler.hooks) {
+      compiler.hooks.compilation.tap('HtmlWebpackHarddisk', compilation => {
+        compilation.hooks.htmlWebpackPluginAfterHtmlProcessing.tapAsync(
+          'HtmlWebpackHarddisk',
```

Comment:
This should also be updated to match the plugin name

Reply:
This has also been updated in the latest commit.